projects/styles/lf-laserfiche-lite/sass/_dropdown-lf.scss

```diff
@@ -18,7 +18,7 @@ mat-label.lf-dropdown-label {
 }
 
 .lf-dropdown .mat-form-field-label-wrapper, .lf-dropdown .mat-form-field-label {
-  top: 0;
+  top: 0 !important;
```

Comment:
!important is not recommended...
weird that that's necessary
is the style that it's overriding more specific? (i.e two classes not just .mat-form-field-label-wrapper)?

Reply:
try added both those classes to the css selector instead of important